apps/src/dance/Dance.js

```diff
@@ -417,6 +418,7 @@ Dance.prototype.onReportComplete = function (response) {
  * Click the run button.  Start the program.
  */
 Dance.prototype.runButtonClick = async function () {
+  document.getElementById('danceClickToRun').style.display = "none";
```

Comment:
Does this break when not in share view?  I'd expect `document.getElementById('danceClickToRun')` to return `undefined` when we're not in share view.

Reply:
Yep, this broke when not in share view. Thanks for looking closely! I updated to only hide/show that element if it exists.

Reply:
does this at all break or remove the need for the loading spinner?